scripts/styles_check.sh

```diff
@@ -3,14 +3,14 @@
 # This script is used to check if the style.css file is up to date.
 
 # Check if the git tree for CWD is clean
-if [ -n "$(git status --porcelain)" ]; then
+if [ -n "$(git status internal/view/assets/less internal/view/assets/css --porcelain)" ]; then
```

Comment:
I believe we only need to check the css path here and below, since that's the generated code.